client/app/src/main/java/com/example/criengine/Database/GoogleBooksWrapper.java

```diff
@@ -0,0 +1,111 @@
+package com.example.criengine.Database;
+
+import android.os.AsyncTask;
+
+import com.example.criengine.Objects.Book;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.concurrent.ExecutionException;
+
+public class GoogleBooksWrapper {
+    String API_KEY = "AIzaSyBtcP5fFkabwkm66RkqF3PnSToFNAw0cdY";
```

Comment:
ideally api keys are handled in some config that's not on github but i think it's fine for what we're doing

Reply:
i can run these queries without the key haha, Im just using it as a courtesy